libs/src/LSPlugin.d.ts

```diff
@@ -0,0 +1,196 @@
+import EventEmitter from 'eventemitter3'
```

Comment:
Is this typing file generated by TSC or manually maintained?

Reply:
Is this file `LSPlugin.d.ts`? Currently it's manually maintained.